R/derive_vars_query.R

```diff
@@ -1,5 +1,8 @@
 #' Derive Query Variables
 #'
+#' **Note:** This function exists to work with derive_vars_query, to execute
+#' the first step of derive_vars_query separately for metadata tracking
```

Comment:
Either we should remove this note or we should explain what is meant by "metadata tracking".

Reply:
@paltusplintus do you have a better way to say this?